### ISSUE TITLE
training model collapse

### ISSUE THREAD

**Issue by wtliao** [royorel/Lifespan_Age_Transformation_Synthesis#15]
Hi, thanks for sharing your nice work and the dataset. I am playing around your code to know more about your idea. However, after training an epoch, the synthesized images of different classes are blank.

after about 1000 iterations
![66fc8bb19dfcc7f563e25487f3a7375](https://user-images.githubusercontent.com/4618954/103356982-68558000-4ab2-11eb-91e3-b8710910532b.png)

after 1 epoch
![44f3b4659a8b8d687f27c1bbd4241b9](https://user-images.githubusercontent.com/4618954/103356942-52e05600-4ab2-11eb-81d1-21997fc043b0.png)

after 5 epoches
![c8a20c97691ff157088a5d1c7953bd7](https://user-images.githubusercontent.com/4618954/103357003-70152480-4ab2-11eb-92d0-72bf93cc06d3.png)
 
Training loss over time
![newplot](https://user-images.githubusercontent.com/4618954/103357404-65a75a80-4ab3-11eb-945d-db044943ce46.png)

I have not changed anything of your code. Do you have any idea about that? Thanks a lot.

**Comment by royorel** [comment]
Hi @wtliao,

This is known and happens from time to time because of the relatively high initial learning rate needed to train the Equalized learning rate StyleGAN convolution blocks. Sometimes this issue resolves itself at later stages of training (It should be solved by epoch 150). Alternatively, you can try lowering the learning rate. It will stabilize the training, but the aging effect might not be the same as in the paper.

**Comment by wtliao** [comment]
Hi @royorel thanks a lot for you swift reply :). I will let it run till 200 epoches and have a look. I have one more question about the batch_size setting. I notice that the default bs=6 on 4 GPUs. I am a little confused that how 6 samples are assigned to 4 GPUs at each iteration?

**Comment by royorel** [comment]
@wtliao each sample is actually a pair of images. So overall it's 12 images over 4 GPUs (3 per GPU)
